### DISCUSSION TITLE
Package listed as prerelease

### DISCUSSION THREAD

**Discussion by Panda-Sharp** [community/community#25031] [Packages]
<p>Hello,</p>
<p>I have to similar workflows and nuspec:</p>
<ol>
<li>
<a href="https://github.com/emiliano84/Toolkit/blob/master/.github/workflows/nugetUwp.yml" rel="nofollow noopener">https://github.com/emiliano84/Toolkit/blob/master/.github/workflows/nugetUwp.yml</a><br>
<a href="https://github.com/emiliano84/Toolkit/blob/master/Yugen.Toolkit.Uwp/Yugen.Toolkit.Uwp.nuspec" rel="nofollow noopener">https://github.com/emiliano84/Toolkit/blob/master/Yugen.Toolkit.Uwp/Yugen.Toolkit.Uwp.nuspec</a>
</li>
<li>
<a href="https://github.com/emiliano84/Toolkit/blob/master/.github/workflows/nugetUwpControls.yml" rel="nofollow noopener">https://github.com/emiliano84/Toolkit/blob/master/.github/workflows/nugetUwpControls.yml</a><br>
<a href="https://github.com/emiliano84/Toolkit/blob/master/Yugen.Toolkit.Uwp.Controls/Yugen.Toolkit.Uwp.Controls.nuspec" rel="nofollow noopener">https://github.com/emiliano84/Toolkit/blob/master/Yugen.Toolkit.Uwp.Controls/Yugen.Toolkit.Uwp.Controls.nuspec</a>
</li>
</ol>
<p>that publish a Nuget package to github</p>
<ol>
<li><a href="https://github.com/emiliano84/Toolkit/packages/48448" rel="nofollow noopener">https://github.com/emiliano84/Toolkit/packages/48448</a></li>
<li><a href="https://github.com/emiliano84/Toolkit/packages/65511" rel="nofollow noopener">https://github.com/emiliano84/Toolkit/packages/65511</a></li>
</ol>
<p>the first one is correctly listed in visual studio, the second one appears only if I tick the “include prerelease” checkbox, can you help me to understand why?</p>
<p>thanks</p>
<p>regards</p>

**Comment by weide-zhou** [comment]
<p>I forked your repo and created both packages, however both packages can display well on my VS2019 without ‘prelease’ checked, could you please have a double check? eg, recreate package and change another vs client?</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181088506-69df1c02-0588-478a-aafc-52270b399417.png" alt="package.png" data-base62-sha1="nfCOlTQRMWlOvSyq8oKQXnuz7qd" width="680" height="292"></p>

**Comment by Panda-Sharp** [comment]
<p><a class="mention" href="/weide-zhou">@weide-zhou</a> thanks to try and let me know, I’ll double check, but I already created a new version of the package several times. </p>
<p>About change another VS client, do you mean a different Visual Studio/PC? not the one I’m using?</p>
<p>thanks</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>I download your nuget packages and checked it can show well on my vs2019. Please check your vs client.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181088508-2fcd1386-ba2c-45d9-8481-e355df290024.png" alt="package2.png" data-base62-sha1="lNg6CxmkvQGmpyJZpvctOC9Gs0c" width="607" height="300"></p>

**Comment by Panda-Sharp** [comment]
<p>I can confirm that on my friends VS worked, thanks :slight_smile:</p>

**Comment by gsitv** [comment]
<p>So what is the solution now? I have the same problem all my packages are only available for installation if pre-release checkbox is ticked.</p>

**Comment by cat-prices** [comment]
<p>I’m getting the same thing. Publishing *Private* packages to a private repo and they only show up if you click the Prerelease checkbox. Glad I found your post as I thought I was going mad.</p>

**Comment by DStastny** [comment]
<p>As above mentioned this is not working for me.  Packages show as prerelease.  When are we going to see UI to manage packages.  GraphQL is not a solution calling raw APIs.</p>

**Comment by sebfischer83** [comment]
<p>Did anyone has a solution? </p>

**Comment by robturner-kaluza** [comment]
<p>There’s some minimal properties that you must specify inside the PropertyGroup tag on your csproj.</p>
<pre><code>&lt;PackageId&gt;AppLogger&lt;/PackageId&gt;
&lt;Version&gt;1.0.0&lt;/Version&gt;
&lt;Authors&gt;your_name&lt;/Authors&gt;
&lt;Company&gt;your_company&lt;/Company&gt;
</code></pre>
<p>In my case I was missing an explicit Version tag even though I was generating one when doing dotnet pack.</p>

**Comment by Panda-Sharp** [comment]
<blockquote>
<p><a class="mention" href="/robturnerovo">@robturnerovo</a> wrote:</p>
</blockquote>
<blockquote>
<p>There’s some minimal properties that you must specify inside the PropertyGroup tag on your csproj.</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>AppLogger<br>
1.0.0<br>
your_name<br>
your_company</p>
</blockquote>
<blockquote>
<p>In my case I was missing an explicit Version tag even though I was generating one when doing dotnet pack.</p>
</blockquote>
<p>I’ll try this solution out, but both of them are without these properties, first one works and second one no</p>
